plugins/kotlin-dataframe/testData/box/propertiesOrder.fir.txt

```diff
@@ -1,6 +1,4 @@
-FILE: read.kt
-    package org.jetbrains.kotlinx.dataframe
-
+FILE: propertiesOrder.kt
     public final fun box(): R|kotlin/String| {
```

Comment:
You wanted to exclude these files right?

Reply:
I opt in fir dump for this file specifically. Would be good to have a few of them present to track changes in generated code, like for example there's this `@Order` annotation and IDE plugin relies on it